src/Bot.ts

```diff
@@ -1,10 +1,10 @@
 import * as builder from "botbuilder";
-import * as config from "config";
 import { RootDialog } from "./dialogs/RootDialog";
 import { SetLocaleFromTeamsSetting } from "./middleware/SetLocaleFromTeamsSetting";
 import { StripBotAtMentions } from "./middleware/StripBotAtMentions";
 // import { SetAADObjectId } from "./middleware/SetAADObjectId";
 import { LoadBotChannelData } from "./middleware/LoadBotChannelData";
+import { ResetBotChat } from "./middleware/ResetBotChat";
```

Comment:
Do we want to name this something indicating its "Fakeness"? Something like "ImitateConversationStart" or something? (Perfectly ok if the answer is no) Also, do we want the name or a note to mention that this really is for 1:1 chats? Thinking about it, I'm assuming if you called this in a channel thread, you would get a memberAdded event for - not the channel.id, but that chat threads id (which could muck with expectations or something).

Reply:
Renamed to SimulateResetBotChat.